### PR TITLE
Connect with us

### PR DESCRIPTION
**Pull request by rbaia28**
Small additions, created recipe page for Chris and linked back on the navbar. Added PHP page for connecting with us form. Currently writing the code. We will need to adjust the file names to PHP because of this but we will hurdle that when we get there

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,9 @@ <h1>Pizza Party
                 <a href="ourstory.html ">Our Story</a>
                 <a href="Menu.html ">Menu</a>
                 <a href="Reservations.html ">Reservations</a>
-                <a href="contactus.html ">Connect with Us!</a>
+                <a href="contactus.php">Connect with Us!</a>
```
Comment (gc824) [inline index.html:21]: do we need two contactus file? 
contactus.html
contactus.php

**Review by gc824 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (rbaia28) [reply index.html:21]: it was removed thats why its in red. i updated it to be a PHP file because it included PHP. There is only one contact us file.

Comment (gc824) [reply index.html:21]: opps! that makes sense.

**Review by gc824 — APPROVED** [review]

**Comment by billythakidd04** [comment]
Was this related to a specific ticket?

**Pull request merged**